modules/ts/src/ts_perf.cpp

```diff
@@ -2104,8 +2104,6 @@ struct KeypointComparator
     {
         return cmp(pts_[idx1], pts_[idx2]);
     }
-private:
-    KeypointComparator& operator=(const KeypointComparator&) = delete;
```

Comment:
Removal is not an option for `= delete`.
Usually it is a guard, because default compiler generated code is not correct and we should not use copying at all.

Try to declare deleted copy ctor.

Reply:
In this specific case I thought that this class is only used in a single place to sort keypoints and nobody actually use `operator=` because it is also `private`. So I decided that probably this was just a habit of implementing safe class.

Maybe it would be better to rewrite `sort` like this? What do you think?
```
    comparators::KeypointGreater cmp;
    std::sort(idxs.data(), idxs.data() + desc.rows, 
        [&](int lhs, int rhs){ return cmp(pts[lhs], pts[rhs]); });
```

Reply:
There is no labmda's on 3.4 branch (no C++11)